test/actions/katello/host/update_system_purpose_test.rb

```diff
@@ -0,0 +1,51 @@
+require 'katello_test_helper'
+
+module Katello::Host
+  class UpdateSystemPurposeTest < ActiveSupport::TestCase
+    include Dynflow::Testing
+    include Support::Actions::Fixtures
+    include FactoryBot::Syntax::Methods
+
+    before :all do
+      User.current = users(:admin)
+      @host = FactoryBot.create(:host, :with_subscription)
+    end
+
+    describe 'update system purpose' do
+      let(:action_class) { ::Actions::Katello::Host::UpdateSystemPurpose }
+
+      it 'plans' do
+        service_level = 'Standard'
```

Comment:
These variables with syspurpose values look duplicated b/t the two tests. consider making them instance variables defined in the  setup method.

Reply:
Updated